src/EnergyPlus/GroundHeatExchangers.cc

```diff
@@ -1311,9 +1289,9 @@ void GLHEVert::calcShortTimestepGFunctions(EnergyPlusData &state)
             } else {
                 // all other cells
 
-                auto &westCell = Cells[cell_index - 1];
-                auto &thisCell = Cells[cell_index];
-                auto &eastCell = Cells[cell_index + 1];
+                auto const &westCell = Cells[cell_index - 1];
+                auto const &eastCell = Cells[cell_index + 1];
+                auto const &thisCell = Cells[cell_index];
```

Comment:
[src/EnergyPlus/GroundHeatExchangers.cc:1314]:(style),[constVariable],Variable 'westCell' can be declared as reference to const
[src/EnergyPlus/GroundHeatExchangers.cc:1315]:(style),[constVariable],Variable 'thisCell' can be declared as reference to const
[src/EnergyPlus/GroundHeatExchangers.cc:1316]:(style),[constVariable],Variable 'eastCell' can be declared as reference to const